x-pack/plugins/ml/public/application/trained_models/models_management/test_models/test_flyout.tsx

```diff
@@ -30,15 +62,41 @@ export const TestTrainedModelFlyout: FC<Props> = ({ model, onClose }) => {
               />
             </h2>
           </EuiTitle>
-        </EuiFlyoutHeader>
-        <EuiFlyoutBody>
+          <EuiSpacer size="s" />
           <EuiTitle size="xs">
             <h4>{model.model_id}</h4>
           </EuiTitle>
+        </EuiFlyoutHeader>
+        <EuiFlyoutBody>
+          <EuiTabs
+            size="m"
+            css={{
+              marginTop: `-${mediumPadding}`,
+            }}
+          >
+            <EuiTab
+              isSelected={inputType === INPUT_TYPE.TEXT}
+              onClick={() => setInputType(INPUT_TYPE.TEXT)}
+            >
+              <FormattedMessage
+                id="xpack.ml.trainedModels.testModelsFlyout.textTab"
+                defaultMessage="Test using text"
+              />
+            </EuiTab>
+            <EuiTab
+              isSelected={inputType === INPUT_TYPE.INDEX}
+              onClick={() => setInputType(INPUT_TYPE.INDEX)}
+            >
+              <FormattedMessage
+                id="xpack.ml.trainedModels.testModelsFlyout.indexTab"
+                defaultMessage="Test using existing Index"
```

Comment:
I think this should be `index` rather than `Index` for sentence case guidelines.

Reply:
Updated in https://github.com/elastic/kibana/pull/144629/commits/2c5e9caf9368e22fed244aceb44fa48d1edb30b4